charts/gluu/openbanking-values.yaml

```
@@ -1,5 +1,26 @@
 # -- OAuth Authorization Server, the OpenID Connect Provider, the UMA Authorization Server--this is the main Internet facing component of Gluu. It's the service that returns tokens, JWT's and identity assertions. This service must be Internet facing.
 auth-server:
+  # -- Add tolerations for the pods
+  tolerations: []
+  # -- Configure the topology spread constraints. Notice this is a map NOT a list as in the upstream API
+  # https://kubernetes.io/docs/concepts/scheduling-eviction/topology-spread-constraints/
+  topologySpreadConstraints: {}
+    # -- Define below as many constraints as needed. The key name should follow the structure tsc1, tsc2...etc.
```

YAMLlint (1.38.0)

[warning] comment not indented like content

(comments-indentation)